website/docs/r/emr_cluster.html.markdown

```diff
@@ -0,0 +1,313 @@
+    ssh_enable = true
+    master_pwd = "ABCtest1234!"
+}
+```
+
+#### 3. Scale Down
+In the case of scaling down a cluster, we need to specified the host group and the instance list. 
+
+The following is an example. We scale down the cluster by decreasing the node count by 2, and specifying the scale-down instance list.
+```
+// Zones data source for availability_zone
+data "alicloud_zones" "default" {
+    available_resource_creation = "emr"
+}
+
+resource "alicloud_vpc" "vpc" {
+    name       = "${var.vpc_name}"
+    cidr_block = "172.16.0.0/16"
+}
+
+resource "alicloud_security_group" "default" {
+    name        = "${var.security_group_name}"
+    vpc_id      = "${alicloud_vpc.vpc.id}"
+}
+
+// VSwitch Resource for Module
+resource "alicloud_vswitch" "vswitch" {
+    availability_zone = "${var.availability_zone}"
+    name              = "${var.vswitch_name}"
+    cidr_block        = "${var.vswitch_cidr}"
+    vpc_id            = var.vpc_id == "" ? alicloud_vpc.vpc[0].id : var.vpc_id
+}
+
+resource "alicloud_emr_cluster" "default" {
+    name = "terraform-resize-test-0923"
+
+    emr_ver = "EMR-3.22.0"
+
+    cluster_type = "HADOOP"
+
+    host_group {
+        host_group_name = "master_group"
+        host_group_type = "MASTER"
+        node_count = "2"
+        instance_type = "ecs.g5.xlarge"
+        disk_type = "CLOUD_SSD"
+        disk_capacity = "80"
+        disk_count = "1"
+        sys_disk_type = "CLOUD_SSD"
+        sys_disk_capacity = "80"
+    }
+
+    host_group {
+        host_group_name = "core_group"
+        host_group_type = "CORE"
+        node_count = "2"
+        instance_type = "ecs.g5.xlarge"
+        disk_type = "CLOUD_SSD"
+        disk_capacity = "80"
+        disk_count = "4"
+        sys_disk_type = "CLOUD_SSD"
+        sys_disk_capacity = "80"
+    }
+
+    host_group {
+        host_group_name = "task_group"
+        host_group_type = "TASK"
+        node_count = "2"
+        instance_type = "ecs.g5.xlarge"
+        disk_type = "CLOUD_SSD"
+        disk_capacity = "80"
+        disk_count = "4"
+        sys_disk_type = "CLOUD_SSD"
+        sys_disk_capacity = "80"
+        instance_list = "[\"instance_id1\", \"instance_id2\"]"
+    }
+
+    high_availability_enable = true
+    option_software_list = ["HBASE","PRESTO",]
+    zone_id = "cn-huhehaote-a"
+    security_group_id = var.security_group_id == "" ? alicloud_security_group.default[0].id : var.security_group_id
+    is_open_public_ip = true
+    charge_type = "PostPaid"
+    vswitch_id = var.vswitch_id == "" ? alicloud_vswitch.vswitch[0].id : var.vswitch_id
+    user_defined_emr_ecs_role = "EMRUserDefineRole-Role1"
+    ssh_enable = true
+    master_pwd = "ABCtest1234!"
+}
+```
+
+
+## Argument Reference
+The following arguments are supported:
+
+* `name` - (Required) The name of emr cluster.
+* `emr_ver` - (Required, ForceNew) EMR Version, e.g. EMR-3.22.0. You can find the all valid EMR Version in emr web console.
```

Comment:
Does the version can be list? If not, is there any docs to show them?

Reply:
No docs.  the version could be find in web console when creating.